x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.test.tsx

```diff
@@ -508,16 +510,26 @@ describe('alerts_list component with items', () => {
     ).toBeTruthy();
 
     let percentiles = wrapper.find(
-      `EuiTableRowCell[data-test-subj="alertsTableCell-ruleExecutionPercentile"] span[data-test-subj="${Percentiles.P50}Percentile"]`
+      `EuiTableRowCell[data-test-subj="alertsTableCell-ruleExecutionPercentile"] span[data-test-subj="rule-duration-format-value"]`
     );
 
     mockedAlertsData.forEach((rule, index) => {
       if (typeof rule.monitoring?.execution.calculated_metrics.p50 === 'number') {
+        // Ensure the table cells are getting the correct values
         expect(percentiles.at(index).text()).toEqual(
-          `${rule.monitoring.execution.calculated_metrics.p50}ms`
+          getFormattedDuration(rule.monitoring.execution.calculated_metrics.p50)
         );
+        // Ensure the tooltip is showing the correct content
+        expect(
+          wrapper
+            .find(
+              'EuiTableRowCell[data-test-subj="alertsTableCell-ruleExecutionPercentile"] [data-test-subj="rule-duration-format-tooltip"]'
+            )
+            .at(index)
+            .props().content
+        ).toEqual(getFormattedMilliseconds(rule.monitoring.execution.calculated_metrics.p50));
       } else {
-        expect(percentiles.at(index).text()).toEqual('N/A');
+        expect(percentiles.at(index).text()).toEqual('00:00');
```

Comment:
I think we should keep the N/A to me it means that we do not have any data yet so it is Not Applicable

Reply:
you can add a test like `expect(getFormattedDuration(null)).toEqual('N/A');